docs/content/doc/installation/from-source.en-us.md

```diff
@@ -38,17 +38,19 @@ Gitea</a>
 
 ## Download
 
-First, retrieve the source code. The easiest way is to use the Go tool. Use the
-following commands to fetch the source and switch into the source directory.
-Go is quite opinionated about where it expects its source code, and simply
-cloning the Gitea repository to an arbitrary path is likely to lead to
-problems - the fixing of which is out of scope for this document.
+First, we must retrieve the source code. Since, the advent of go modules, the
+simplest way of doing this is to use git directly as we no longer have to have
+gitea built from within the GOPATH. 
 
 ```bash
-go get -d -u code.gitea.io/gitea
-cd "$GOPATH/src/code.gitea.io/gitea"
+# From within your src directory
+cd "$HOME"/src
+git clone https://github.com/go-gitea/gitea
```

Comment:
What about using
GOBIN=/usr/local/bin go install -v -tags 'bindata sqlite' -ldflags '-s -w' code.gitea.io/gitea
or
GOBIN=. go install -v -tags 'bindata sqlite' -ldflags '-s -w' code.gitea.io/gitea

Reply:
I know I am missing a flag for SQLite but I am on my phone and don't recall it by name :smile:

Reply:
Ok this will not work since it would need bindata generated and if not bindata the file will need to be on fs.

Reply:
I think we could achieve this by setting up a repo (with version) of generated asset and override it by go.mod. I will try something.